blogposts/announcing-sourcegraph-3.5.md

```diff
@@ -0,0 +1,228 @@
+
+The `orgs` field is a list of GitHub organizations:
+
+```
+“orgs”: [
+    “gorilla”
+]
+```
+
+If filtering the list of repositories is required, e.g. `archived:no  forked:no` the `repositoryQuery` should be used instead of `orgs`. If the result of a query entry in `repositoryQuery` exceeds 1,000 repositories, it will need to be split out over multiple entries. Please [contact support](mailto:support@sourcegraph.com) if you require assistance.
+
+![](/blog/3.5-org-settings.png)
+
+![](/blog/3.5-org-repos.png)
+
+## 3.5 changelog
+
+### Added
+
+- Indexed search now supports matching consecutive literal newlines, with queries like e.g. `foo\nbar.*` to search over multiple lines. [#4138](https://github.com/sourcegraph/sourcegraph/issues/4138)
+- The `orgs` setting in [GitHub external service config](https://docs.sourcegraph.com/admin/external_service/github) allows admins to select all repositories from the specified organizations to be synced.
+- A new experimental search filter `repohascommitafter:"30 days ago"` allows users to exclude stale repositories that don't contain commits (to the branch being searched over) past a specified date from their search query.
+- The `authorization` setting in the [Bitbucket Server external service config](https://docs.sourcegraph.com/admin/external_service/bitbucket_server#permissions) enables Sourcegraph to enforce the repository permissions defined in Bitbucket Server.
+- A new, experimental status indicator in the navigation bar allows admins to quickly see whether the configured repositories are up to date or how many are currently being updated in the background. You can enable the status indicator with the following site configuration: `"experimentalFeatures": { "statusIndicator": "enabled" }`.
+
+### Changed
+
+- The saved searches UI has changed. There is now a Saved searches page in the user and organizations settings area. A saved search appears in the settings area of the user or organization it is associated with.
+
+### Removed
+
+### Fixed
+
+- Fixed repository search patterns which contain `.*`. Previously our optimizer would ignore `.*`, which in some cases would lead to our repository search excluding some repositories from the results.
+- Fixed an issue where the Phabricator native integration would be broken on recent Phabricator versions. This fix depends on v1.2 of the [Phabricator extension](https://github.com/sourcegraph/phabricator-extension).
+- Fixed an issue where the "Empty repository" banner would be shown on a repository page when starting to clone a repository.
+- Prevent data inconsistency on cached archives due to restarts. (#4366)
+- On the /extensions page, the UI is now less ambiguous when an extension has not been activated. (#4446)
+
+
+The [changelog for this and previous releases](https://github.com/sourcegraph/sourcegraph/blob/master/CHANGELOG.md#3.5) is available on GitHub.
```

Comment:
I don't think it's important to say "and previous releases". It's true, but why does it matter? It also sounds awkward ("the changelog for this ... releases"). And why say "is available on GitHub"? That sounds like filler and needlessly mentions GitHub here.

```suggestion
See the [full Sourcegraph changelog](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/CHANGELOG.md).
```

Reply:
Great suggestion!